src/hooks/tests/use-cart.spec.js

```diff
@@ -0,0 +1,109 @@
+import { useDispatch } from 'react-redux';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useCart } from '../use-cart';
+import { mockItem, mockPromoCode, sizeAndPrice } from './use-cart.variables';
+
+const dispatch = jest.fn();
+
+jest.mock('react-redux');
+
+useDispatch.mockImplementation(() => dispatch);
+
+describe('use-cart tests', () => {
+  it('should add item to cart', () => {
+    const { result } = renderHook(useCart);
+    act(() => {
+      result.current.cartOperations.addToCart(mockItem);
+    });
+
+    expect(result.current.cart).toContain(mockItem);
+  });
+  it('should return cart item by id', () => {
+    const { result } = renderHook(useCart);
```

Comment:
this line duplicates in each test, maybe move it to beforeEach

Reply:
it will be not working because of scope of variable, and if i put it just in describe scope the state would not change

Reply:
i have left it from my attention, thanks

Reply:
done